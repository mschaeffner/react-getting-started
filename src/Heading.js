import React, { Component } from 'react';

class Heading extends Component {

  render() {
    return (
      <h1>Friend list ({this.props.count})</h1>
    );
  }

}

export default Heading;
