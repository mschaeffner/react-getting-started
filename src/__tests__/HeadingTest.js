import React from 'react';
import renderer from 'react-test-renderer';
import Heading from '../Heading';


describe('Heading', () => {
  it('should show the total number of friends', () => {

    const component = renderer.create(
      <Heading count={75} />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

  });
});
