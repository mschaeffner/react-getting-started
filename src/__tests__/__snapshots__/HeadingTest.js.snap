// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should show the total number of friends 1`] = `
<h1>
  Friend list (
  75
  )
</h1>
`;
